g3doc/quick_reference.md

```diff
@@ -548,6 +548,24 @@ from left to right, of the arguments passed to `Create{2-4}`.
 
 *   <code>V **AbsDiff**(V a, V b)</code>: returns `|a[i] - b[i]|` in each lane.
 
+*   <code>V **PairwiseAdd**(D d, V a, V b)</code>: Add consecutive pairs of elements.
+    Return the results of a and b interleaved, such that `r[i] = a[i] + a[i+1]` for
+    even lanes and `r[i] = b[i-1] + b[i]` for odd lanes.
+
+*   <code>V **PairwiseSub**(D d, V a, V b)</code>: Subtract consecutive pairs of elements.
+    Return the results of a and b interleaved, such that `r[i] = a[i+1] - a[i]` for
+    even lanes and `r[i] = b[i] - b[i-1]` for odd lanes.
+
+*   <code>V **PairwiseAdd128**(D d, V a, V b)</code>: Add consecutive pairs of
```

Comment:
Please move the *128 ops into the "### Blockwise" section of this doc; all ops with such behavior are gathered there.
One other thing to consider: for consistency, we could name the current PairwiseAdd PairwiseAddWhole, as with InterleaveLower and InterleaveWholeLower, and this one PairwiseAddBlocks or similar?
@johnplatts , any preference?